### PR TITLE
feat: show comments in popup for each movie

### PR DESCRIPTION
**Pull request by YKerroum**
- [x] Show comments in the pop-up of each movie.
- [x] Make a new module for the comments functions.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,6 +29,11 @@ list.addEventListener('click', (e) => {
   if (e.target.classList.contains('comment')) {
     const movieId = parseInt(e.target.id.slice(-1), 10);
     displayPopUp(movies.find((m) => m.id === movieId));
+    // const popupComments= document.getElementById('popComment');
+    // const popupHead = document.getElementById('commentCounter');
+    // if(popupComments.childElementCount===0) {
+    // popupHead.innerHTML = 'Comments(0)';
+    // };
```
Comment (sebalp100) [inline src/index.js:36]: Kindly remove any comments that are not necessary for your code to run properly. That way you can make your code look more organized and professional 👍

**Review by sebalp100 — CHANGES_REQUESTED** [review]
♻️ STATUS: REQUIRED CHANGES ♻️
Hi @YKerroum  👋,
Good job so far!

so-close-so-close-seth-meyers


There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights 🚀
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Comments displaying as expected

Required Changes ♻️
Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/index.js (hunk 1 above)
Comment (YKerroum) [reply src/index.js]: Done, thanks for this suggestion.

**Review by sebalp100 — APPROVED** [review]
STATUS: APPROVED 🟢

ok


Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name
✔️ Required changes implemented

Great implementation of all the requirements in this milestone. 👏

You have done an amazing job 👍

Your project is complete! There is nothing else to say other than... it's time to merge it. :shipit:

Congratulations! 🎉

CHEERS AND HAPPY CODING!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**